tools/rosbash/rosbash

```diff
@@ -1026,4 +1081,5 @@ complete -F "_roscomplete_rossrv" "rossrv"
 complete -F "_roscomplete_roscreate_pkg" "roscreate-pkg"
 complete -F "_roscomplete_roswtf" -o filenames "roswtf"
 complete -F "_roscomplete_rosconsole" "rosconsole"
-complete -F "_roscomplete_file" "rosmv"
+complete -F "_roscomplete_filemv" "rosmv"
+
```

Comment:
Still an extra empty line.

Reply:
I was under the impression that we leave one space at the end of the file. :)
Thank you for pointing out.

Reply:
A file should end with a newline. But not with two or more.

If your editor doesn't show the already present newline I think it is time for a new one ;-)